lms/envs/common.py

```diff
@@ -2146,6 +2146,9 @@
     INSTALLED_APPS += ('class_dashboard',)
 
 ################ Enable credit eligibility feature ####################
+FEATURES['ENABLE_JWT_AUTH'] = False
```

Comment:
the comment above this is wrong.
it should be spelled out here what features are going to be broken if this is not set to `True`.